### DISCUSSION TITLE
How could I hide the console when running mitmdump

### DISCUSSION THREAD

**Discussion by yyyrrr1997** [mitmproxy/mitmproxy#5303] [Q&A] [answered]
Dear developers,I have packaged my project by pyinstaller ,and there is a command in my code:
![image](https://user-images.githubusercontent.com/64613345/165667217-a6b96782-5baf-4556-8c95-98e266dba4d2.png)

The problem is when I start the exe, there is always a mitmdump console show out.
How could I hide this console for a better user experience.
![image](https://user-images.githubusercontent.com/64613345/165668466-e50ec4f8-daaf-4f05-b07c-ca1dd33a5551.png)

Hope for your help,Thanks!

**Comment by Prinzhorn** [comment, marked as answer]
This is not related to mitmproxy. I'm not a Python expert, DuckDuckGo brings up things like this https://stackoverflow.com/questions/25122823/hide-console-with-subprocess-popen

**Reply by yyyrrr1997** [reply]
It works ,thank you !
